app/src/gui/components/record/relationships/RelatedInformation.tsx

```diff
@@ -897,14 +990,17 @@ export async function addRecordLink(
       new_doc['deleted'] = latest_record.deleted;
       await upsertFAIMSData(child_record.project_id, new_doc);
     }
-    // here we are trusting that Record has enough of the fields of RecordMetadata
-    // for the purposes of the caller until such time as we rationalise the Record types
+    // here we are trusting that Record has enough of the fields of
+    // RecordMetadata for the purposes of the caller until such time as we
+    // rationalise the Record types
     child_record_meta = latest_record as unknown as RecordMetadata;
     // it's missing the HRID so grab it here
-    child_record_meta.hrid = getHRIDValue(
-      child_record_meta.record_id,
-      child_record_meta.data
-    );
+    // Just use record id here and see what happens ¯\_(ツ)_/¯
```

Comment:
What happens!? I'm dying to know!

Reply:
Child links seem to be fine.

Reply:
Ah no, so this populates the list of records if you select 'add link to an existing XXX', we now see a list of record IDs rather than the HRIDs. So we really want to have the HRID here.